kernels/portable/cpu/op_full.cpp

```diff
@@ -38,7 +38,8 @@ Tensor& full_out(
 
   ET_SWITCH_SCALAR_OBJ_TYPES(val_type, ctx, name, CTYPE_VAL, [&] {
     CTYPE_VAL val;
-    utils::extract_scalar(fill_value, &val);
+    ET_KERNEL_CHECK(
```

Comment:
Had to adjust due to errors like this: 

https://github.com/pytorch/executorch/actions/runs/13123283587/job/36614101657?pr=8016

Reply:
I don't think ET_KERNEL_CHECK will do the right thing in a lambda -- it returns from the lambda, not the operator

Reply:
any suggestions on how to fix? 

i think it's fine, since it will return void from the lambda, but sets context to be fail, with error code InvalidArgument.